client/me/sidebar/style.scss

```diff
@@ -14,3 +14,42 @@
 .is-group-me .sidebar__menu-happychat {
 	display: none;
 }
+
+.sidebar {
+	.sidebar__menu-icon {
+		height: 20px;
+		width: 20px;
```

Comment:
I wasn't expecting us to change the non-collapsed icon sizes, but I think it's a fair change. It makes them consistent with the icons in the My Sites sidebar.

Reply:
> I wasn't expecting us to change the non-collapsed icon sizes, but I think it's a fair change. It makes them consistent with the icons in the My Sites sidebar.

I wasn't expecting to either, but it does make the whole experience much nicer.